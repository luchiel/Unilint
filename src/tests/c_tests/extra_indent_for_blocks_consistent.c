//this one should be ok with either indented = true or false
int zubr() {}

int abaca()
    {
        //indented block
        return 0;
    }

int ababaca()
    {
        //non-indented block
        int z = 0;
        if(z)
            {
                zubr();
            }
        else
        {
            abaca();
        }

        if(z)
            z += z;

        if(z) {
                //indented block
                z++;
            }
    }