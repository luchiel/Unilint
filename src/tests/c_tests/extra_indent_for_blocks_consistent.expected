19:1: indent too small
20:1: indent too small
21:1: indent too small
